tests/conftest.py

```
@@ -0,0 +1,40 @@
+    monkeypatch.setenv("MLFLOW_TRACKING_URI", uri)
+    monkeypatch.setenv("MLFLOW_REGISTRY_URI", uri)
+    monkeypatch.setenv("PROMPT_REGISTRY_ACTIVE_ALIAS", "active")
+    monkeypatch.setenv("PROMPT_REGISTRY_AUTO_SEED", "true")
+
+    try:
+        import prompts
+
+        prompts.registry._initialized = False
+        prompts.registry._client = None
+    except Exception:
+        pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)